.projenrc.ts

```diff
@@ -276,12 +272,22 @@ const repoProject = new yarn.Monorepo({
 
   buildWorkflowOptions: {
     preBuildSteps: [
+      {
+        name: 'Nx cache',
+        uses: 'actions/cache@v4',
+        with: {
+          'key': '${{ runner.os }}-nxcache',
+          'path': ['.nx/cache'].join('\n'),
```

Comment:
I'm not sure this is safe. I regularly need to blow the nx cache away in order to get things to builder properly.